<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sheetflow_main</class>
 <widget class="QMainWindow" name="sheetflow_main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>sheetflow_main</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="file_new_action"/>
    <addaction name="file_open_action"/>
    <addaction name="file_save_action"/>
    <addaction name="fiel_other_action"/>
    <addaction name="printf_action"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>窗口</string>
    </property>
    <addaction name="display_draw_action"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>编辑</string>
    </property>
    <addaction name="zoom_in_action"/>
    <addaction name="zoom_out_action"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="file_new_action">
   <property name="icon">
    <iconset>
     <normaloff>image/filenew.png</normaloff>image/filenew.png</iconset>
   </property>
   <property name="text">
    <string>新建</string>
   </property>
   <property name="statusTip">
    <string>新建</string>
   </property>
  </action>
  <action name="file_open_action">
   <property name="icon">
    <iconset>
     <normaloff>image/fileopen.png</normaloff>image/fileopen.png</iconset>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
   <property name="statusTip">
    <string>打开</string>
   </property>
  </action>
  <action name="file_save_action">
   <property name="icon">
    <iconset>
     <normaloff>image/filesave.png</normaloff>image/filesave.png</iconset>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
   <property name="statusTip">
    <string>保存</string>
   </property>
  </action>
  <action name="fiel_other_action">
   <property name="icon">
    <iconset>
     <normaloff>image/fileother_save.png</normaloff>image/fileother_save.png</iconset>
   </property>
   <property name="text">
    <string>另存为</string>
   </property>
   <property name="statusTip">
    <string>另存为</string>
   </property>
  </action>
  <action name="printf_action">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>image/project.png</normaloff>image/project.png</iconset>
   </property>
   <property name="text">
    <string>打印</string>
   </property>
   <property name="statusTip">
    <string>打印</string>
   </property>
  </action>
  <action name="zoom_in_action">
   <property name="icon">
    <iconset>
     <normaloff>image/zoom_in.png</normaloff>image/zoom_in.png</iconset>
   </property>
   <property name="text">
    <string>放大</string>
   </property>
   <property name="statusTip">
    <string>放大</string>
   </property>
  </action>
  <action name="zoom_out_action">
   <property name="icon">
    <iconset>
     <normaloff>image/zoom_out.png</normaloff>image/zoom_out.png</iconset>
   </property>
   <property name="text">
    <string>缩小</string>
   </property>
   <property name="statusTip">
    <string>缩小 </string>
   </property>
  </action>
  <action name="display_draw_action">
   <property name="icon">
    <iconset>
     <normaloff>image/show_drawer.png</normaloff>image/show_drawer.png</iconset>
   </property>
   <property name="text">
    <string>显示抽屉</string>
   </property>
   <property name="statusTip">
    <string>显示抽屉</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
